presto-benchmark-driver/src/main/java/com/facebook/presto/benchmark/driver/RegexTemplate.java

```diff
@@ -61,8 +60,8 @@ public RegexTemplate(String template)
         try {
             namedGroups = (Map<String, Integer>) NAMED_GROUPS_METHOD.invoke(pattern);
         }
-        catch (Exception e) {
-            throw Throwables.propagate(e);
+        catch (ReflectiveOperationException e) {
+            throw new RuntimeException(e);
```

Comment:
I agree, though it's better to avoid throwing `Error` from a normal method (as opposed to class initialization which you don't normally try to handle errors from)